pom.xml

```diff
@@ -93,6 +93,11 @@
             <version>${postgis-jdbc.version}</version>
         </dependency>
 
+        <dependency>
```

Comment:
The dependabot PR was failing because of the missing validation implementation jar.